src/components/AircrackNG/AircrackNG.tsx

```diff
@@ -35,8 +35,8 @@ const description = "Aircrack-ng is a tool for cracking WEP and WPA/WPA2 passphr
 const steps =
     "How to use Aircrack-ng:\n\n" +
     "Step 1: Type in the path to wordlist(s) filename(s) including the extension .txt. e.g. 'example.txt'.\n" +
-    "For files containing hexadecimal values, you must put a “h:” in front of the file name.\n" +
-    "Step 2: Select the target network based on the access point's MAC address.\n" +
+    "For files containing hexadecimal values, you must put a “h:” in front of the file name.\n";
```

Comment:
Looks like the edit made to this line, changing `+` to `;` breaks the displayed steps in the user guide:
<img width="1024" alt="image" src="https://github.com/user-attachments/assets/ed135153-6a78-4556-99f5-e1c68f2b45ee">

The semicolon (;) terminates the assignment of the `steps` variable, so the remaining steps are not included in this assignment. 

Can you change this back to a `+` to include the remaining steps?

Reply:
Noting also that on the following line where step 2 is written, you'll need to remove the leading `+` to follow the format of the other sentences.

Reply:
Yeah, this wasn't something that I edited, running npx prettier made this change and I'm not sure why but I should have checked it after.

Reply:
It should be fixed now.